android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -158,8 +159,12 @@ AuthStateHolder createAuthStateFromStoredCredentials() {
         }
         return null;
     }
-
-
+    
+    @Override
+    public void onChange(UserSessionInfo userSessionInfo) {
+        accountDAO.setUserSessionInfo(userSessionInfo);
```

Comment:
This allows us to hook into UserSessionInterceptor in rest-client, which extracts UserSessionInfo sent from Bridge, whether in response body or error body in case of a 412